addons/product/pricelist.py

```diff
@@ -492,7 +476,7 @@ def _check_margin(self, cr, uid, ids, context=None):
               "Expressed in the default UoM of the product."
             ),
         'sequence': fields.integer('Sequence', required=True, help="Gives the order in which the pricelist items will be checked. The evaluation gives highest priority to lowest sequence and stops as soon as a matching item is found."),
-        'base': fields.selection(_price_field_get, 'Based on', required=True, size=-1, help="Base price for computation."),
+        'base': fields.selection(_price_field_get, 'Based on', required=True, size=-1, help='Base price for computation. \n Public Price: The base price will be the Sale/public Price. \n Supplier Section on Product or Cost Price : The base price will be the supplier price if it is set, otherwise it will be the cost price. \n Other Pricelist : Computation of the base price based on another Pricelist.'),
```

Comment:
@rim-odoo 
I have added comment on 'base' field "size=-1" and rebase the branch